src/olympia/blocklist/models.py

```diff
@@ -325,3 +325,24 @@ def save_to_blocks(self):
             self.save()
 
         return blocks
+
+
+class KintoImport(ModelBase):
+    OUTCOME_INCOMPLETE = 0
+    OUTCOME_BADGUID = 1
+    OUTCOME_NOTFIREFOX = 2
+    OUTCOME_BLOCK = 3
+    OUTCOME_REGEXBLOCKS = 4
+    OUTCOME_NOMATCH = 5
+    OUTCOMES = {
```

Comment:
It seems unfortunate that this entire mapping (i.e., the integers to their corresponding string values) has to be duplicated in the migration. I assume that's necessary, and there's no way to import the values from this file into the migration?

Reply:
Unfortunately not - in a safe way anyway.  Django migrations are automatically generated and aren't really supposed to reference much outside the migration (apart from django Models and Fields) so they can be trusted to reliably work in the future when the constants might have changed.

Reply:
Also, when you update the constant this is how django know a new migration is needed to bring the database up to date.

Reply:
Ah, I forgot that migrations are just auto-generated files. Nevermind!